study.md

```diff
@@ -57,5 +57,19 @@ Starting with a client making a GET request to a particular URL, describe how
 the parts of Rails interact to produce and send a response.
 
 ```md
-<!-- your answer here -->
+GET
+REQUEST
+A client will make a GET request to a certain URL.
+Hits the web server
+then it hits the router
+The router will then pass the request to the dispatcher
+From the dispatcher it goes to the controller
+the controller will the assign the request to the model.
+The model will then conduct some backend logic
+retrieve the item from the DB
+RESPONCE
+item gets passes from DB to Controller
```

Comment:
The controller uses the return value of a model method called in the controller action to build a response. You're pretty close here.